promdens.py

```diff
@@ -29,36 +28,34 @@
 
 DESC = "Promoted density approach for initial conditions"
 
+
+### functions and classes ###
 def print_header():
-    print(
-        "\n##########################################################\n"
-        f"###  {DESC}  ###\n"
-        "###                   * * * * *                        ###\n"
-        "###       version 1.0         Jiri Janos 2024          ###\n"
-        "##########################################################\n"
-    )
+    print("\n##########################################################\n"
```

Comment:
Was this autoformatted by your editor? It might be nice to run an autoformatter in the CI. (but that's for later)